crates/language/src/target_language.rs

```diff
@@ -129,6 +129,48 @@ impl PatternLanguage {
         Self::from_string(lang, flavor.as_deref())
     }
 
+impl PatternLanguage {
+    pub fn from_string_or_alias(name: &str, flavor: Option<&str>) -> Option<Self> {
```

Comment:
@xonx4l This should be removed, you don't need both.

Reply:
Done!!!